charts/fint-altinn-download-queue-service/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "fint-altinn-download-queue-service.fullname" . }}-test-connection"
+  labels:
+    {{- include "fint-altinn-download-queue-service.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "fint-altinn-download-queue-service.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
Should it be fint-altinn-download-queue-service.fullname or fint-altinn-download-queue-service.name?

Reply:
Removed test